doc/quickstart.md

```diff
@@ -0,0 +1,98 @@
+# A Quickstart Guide to Proper Benchmarking with BenchExec
+
+This guide provides a brief summary of instructions to set up reliable
+benchmark measurements using BenchExec and important points to consider. It is
+meant for users who either want to set up benchmarking from scratch or already
+have a simple setup using, e.g., `time`, `timeout`, `taskset`, `ulimit`, etc.
+and will guide you how to use `runexec` as a simple but much more reliable
+"drop-in" replacement for these tools.
+
+## Guiding Example
+
+> [!IMPORTANT]
+> If your current setup looks similar to the below example (or you are thinking
+> about such a setup), we strongly recommend following this guide for a much
+> more reliable process.
+
+As an example, suppose that you want to measure the performance of your
+tool `program` with arguments `--foo` and `--bar` on the input files
+`input_1.in` to `input_9.in`. To measure the runtime of the tool, one may run
+```
+$ /usr/bin/time program --foo input_1.in
+$ /usr/bin/time program --bar input_1.in
+$ /usr/bin/time program --foo input_2.in
+...
+```
+etc. and note the results. In case resource limitations are desired (e.g. 
+limiting to 1 CPU and 60 sec of wallclock time), the calls might be
+```
+$ taskset -c 0 timeout 60s /usr/bin/time program ...
```

Comment:
I wonder whether we should show negative examples, and in particular this prominently (as highly visible listings on the top of the file, with copy-and-paste buttons from GitHub, etc.). I fear that some users will stop reading here...

If you think these add value, how about having just 1-2 command lines, and inline with single backticks instead of as blocks, such that they are less strongly highlighted?

Reply:
Fair point, will change